sample/src/main/java/com/uber/rave/sample/RaveActivity.java

```diff
@@ -37,21 +47,88 @@ void callGitHub() {
                 .enqueue(new Callback<Repo>() {
                     @Override
                     public void onResponse(Call<Repo> call, Response<Repo> response) {
-                        makeToast("Call passed Rave validation!");
+                        makeToast("Call passed RAVE validation!");
                     }
 
                     @Override
                     public void onFailure(Call<Repo> call, Throwable t) {
                         Throwable cause = t.getCause();
                         if (cause instanceof RaveException) {
-                            makeToast("Call did not pass Rave validation: " + cause.getMessage());
+                            makeToast("Call did not pass RAVE validation: " + cause.getMessage());
                         } else if (cause != null) {
                             makeToast("Call failed for unknown reason: " + cause.getMessage());
                         }
                     }
                 });
     }
 
+    @OnClick(R.id.storage_request)
+    void fetchFromStorage() {
```

Comment:
Nice! Can we wrap this up in an API and provide it via DI the same way Retrofit is done?